libqtile/group.py

```diff
@@ -251,7 +251,7 @@ def info(self):
             screen=self.screen.index if self.screen else None,
         )
 
-    def add(self, win, focus=True, force=False):
```

Comment:
I assume that `focus` is not passed by another calls in the codebase?

Reply:
I haven't found any, no:
```
$ grep -ir 'add(' libqtile/ test/ | grep focus
libqtile/widget/prompt.py:        hook.subscribe.group_window_add(prevent_focus_steal)
test/test_window.py:        hook.subscribe.group_window_add(set_steal_focus)
```